ui/src/app/cluster-workflow-templates/cluster-workflow-template-details.tsx

```diff
@@ -83,70 +139,41 @@ export function ClusterWorkflowTemplateDetails({history, location, match}: Route
                     {title: name, path: uiUrl('cluster-workflow-templates/' + name)}
                 ],
                 actionMenu: {
-                    items: [
-                        {
-                            title: 'Submit',
-                            iconClassName: 'fa fa-plus',
-                            disabled: edited,
-                            action: () => setSidePanel(true)
-                        },
-                        {
-                            title: 'Update',
-                            iconClassName: 'fa fa-save',
-                            disabled: !edited,
-                            action: () => {
-                                services.clusterWorkflowTemplate
-                                    .update(template, name)
-                                    .then(setTemplate)
-                                    .then(() =>
-                                        notifications.show({
-                                            content: 'Updated',
-                                            type: NotificationType.Success
-                                        })
-                                    )
-                                    .then(() => setError(null))
-                                    .then(() => setEdited(false))
-                                    .catch(setError);
-                            }
-                        },
-                        {
-                            title: 'Delete',
-                            iconClassName: 'fa fa-trash',
-                            action: () => {
-                                popup.confirm('confirm', 'Are you sure you want to delete this cluster workflow template?').then(yes => {
-                                    if (yes) {
-                                        services.clusterWorkflowTemplate
-                                            .delete(name)
-                                            .then(() => navigation.goto(uiUrl('cluster-workflow-templates')))
-                                            .then(() => setError(null))
-                                            .catch(setError);
-                                    }
-                                });
-                            }
-                        }
-                    ]
+                    items: getItems()
```

Comment:
ClusterWorkflowTemplate, CronWorkflow, and WorkflowTemplate all have the same configuration as much as possible.